lua/starfall/libs_cl/render.lua

```diff
@@ -169,6 +169,9 @@ local renderhooks = {
 	renderscene = prepareRenderScene,
 	predrawopaquerenderables = prepareRender,
 	postdrawopaquerenderables = prepareRender,
+	vrprerender = prepareRender,
```

Comment:
good catch there

Reply:
I've tested these hooks, they seem to run but not render anything? or i'm doing something wrong, or it has a matrix or something built into it, i'm not sure, someone should test these hooks to see wether/where they render. using postdrawopaquerenderables is fine too though for VR

Reply:
You tried just a simple render.drawRect?

Reply:
Probably need a code example from the  vrmod library itself.

Reply:
> You tried just a simple render.drawRect?

I'll try to see what that does

> Probably need a code example from the vrmod library itself.

There is none of that i know

Reply:
> You tried just a simple render.drawRect?

Didn't see anything, but the hooks do run every frame like intended.

Reply:
removed the obsolete hook